### PR TITLE
remove kazoo

### PR DESCRIPTION
**Pull request by qiaozhanwei**
1，install.sh remove kazoo
2，MonitorServer and script 
3，RemoveZKNode and script

### CODE REVIEW

File: dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/monitor/ZKMmonitorImpl.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * zk monitor server impl
+ */
+@Component
+public class ZKMmonitorImpl extends AbstractMonitor {
```
Comment (Technoboy-) [inline dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/monitor/ZKMmonitorImpl.java]: ZKMonitorImpl

File: dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/monitor/RunConfig.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Component;
+
+/**
+ * zookeeper conf
+ */
+@Component
+@PropertySource("classpath:config/run_config.conf")
+public class RunConfig {
```
Comment (Technoboy-) [inline dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/monitor/RunConfig.java]: How about naming MonitorConfig

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java [hunk 3]
```diff
@@ -36,6 +36,12 @@
      */
     public static final String HADOOP_PROPERTIES_PATH = "/common/hadoop/hadoop.properties";
 
+
+    /**
+     * run config
+     */
+    public static final String RUN_CONFIG = "/config/run_config.conf";
```
Comment (Technoboy-) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java]: How about naming monitor_config.conf

File: dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/monitor/ZKMmonitorImpl.java (hunk 1 above)
Comment (dailidong) [reply dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/monitor/ZKMmonitorImpl.java]: > ZKMonitorImpl

+1

Comment (qiaozhanwei) [reply dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/monitor/ZKMmonitorImpl.java]: OK

File: dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/monitor/RunConfig.java (hunk 2 above)
Comment (qiaozhanwei) [reply dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/monitor/RunConfig.java]: install.sh write conf file， not monitor config , monitor server only read

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java (hunk 3 above)
Comment (qiaozhanwei) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java]: this not use，I will remove

**Pull request merged**